Assets/ObjectOwnerState.cs

```diff
@@ -6,7 +6,7 @@ public class ObjectOwnerState : MonoBehaviour {
     public Player player;
 
     // Rendering Component contains Material Color
-    private Renderer rend;
+    public Renderer rend;
```

Comment:
I'd say keep this private and add a setter instead.

Reply:
Or you know what, this variable is not needed at all like you said. Just use `GetComponent<Renderer>();` when you need to.